src/core/indexedlist.h

```diff
@@ -44,34 +57,32 @@ class IndexedList
     {
         return v.begin();
     }
-    inline const_iterator begin() const
-    {
-        return v.begin();
-    }
```

Comment:
@BiTOk @sudden6 @Diadlo FYI

Removed the overload completely: `cbegin` is a c++11 feature, which removes ambiguity. As we already set C++11 as a minimum requirement, we don't need old method overloads, which are just being kept for compatibility.